.github/workflows/sync_hf_space.yaml

```diff
@@ -0,0 +1,41 @@
+name: Sync to Hugging Face Space
+
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+jobs:
+  sync-space:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - run: git clone https://${{ secrets.HF_USERNAME }}:${{ secrets.HF_TOKEN }}@huggingface.co/spaces/mozilla-ai/transcribe hf-space
```

Comment:
I don't see this space, have you created it?

Reply:
yes, but it was private. just made it public. for some reason i am having trouble now with installing packages...
https://huggingface.co/spaces/kostissz/transcribe?logs=container

Reply:
> for some reason i am having trouble now with installing packages... https://huggingface.co/spaces/kostissz/transcribe?logs=container

You selected the `gradio` sdk in https://huggingface.co/spaces/kostissz/transcribe/blob/main/README.md but you this code is setting the repo to use the `docker` sdk.
The docker sdk is used in the others because they need GPU.
If this space won't use GPU, then you need to update the code to follow https://huggingface.co/docs/hub/spaces-sdks-gradio#add-the-dependencies

Reply:
ah good catch, thanks! I ll update

Reply:
Thanks for the resources. I adjusted to use Gradio + CPU. Its a bit slow, so I ll probably make another PR soon for GPU when we get the approval to pay for the GPU on HF

Reply:
> GPU when we get the approval to pay for the GPU on HF

We already have it enabled

Reply:
Well I simply just changed from CPU to GPU, and it seems it works just fine with the Gradio SDK and no other code changes!
https://huggingface.co/spaces/mozilla-ai/transcribe